docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -40,6 +47,12 @@ following steps:
 
    Once control plane Pods are up and running, the `kubeadm init` sequence can continue.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
+   completes kubelet dynamic configuration by creating a ConfigMap, RBAC rules for enabling
```

Comment:
>completes kubelet dynamic configuration by creating a ConfigMap, RBAC rules for enabling kubelet access to it,

it completes the kubelet dynamic configuration by creating a ConfigMap and some RBAC rules that enable kubelets to access it,